en/docs/tutorials/integrating-services-tutorial.md

```diff
@@ -0,0 +1,795 @@
+# Integrating Services
+
+In this tutorial, you can learn how to integrate multiple services via Choreo. Here, let's consider a simple example where an online shoe store uses an application that captures online orders. For each order, the application also needs to generate a notification for the sales manager as well as update the inventory recordsinventory records and notify the status to the client who places the order.
+
+## Before you begin
+
+The following are required to try out this tutorial:
+
+- A Google account. In this Google account, open the Google Drive and create a Google sheet named `inventory` with the following table:
+
+    ![Google Sheet Extract](../assets/img/tutorials/inventory-service-google-sheet-extract.png){.cInlineImage-half}
+    
+    | **item**    | **count** |
+    |-------------|-----------|
+    | trainers    | 100       |
+    | loafers     | 80        |
+    | kittenheels | 60        |
+    | maryjanes   | 40        |
+                            
+- A Twilio account
+
+## Step 1: Create a service to manage inventory records
+
+In this step, you are creating a service that does the following:
+
+- Reading order requests captured by the `orders` service that you will create in step 3.
+- Determining whether the order request can be met by checking the inventory records in the `inventory` Google sheet.
+- Generating a response for each order request based on the status (i.e., whether the requested item is available and whether the available quantity is adequate to match the quantity requested).
+
+To create this service, follow the procedure below:
+
+1. Access the Choreo Console and click **Services**. Then click **Create**.
+
+2. Enter `inventory` as the name and click **Create**.
+
+3. To define the API that triggers your service, click **PUT**, enter `/inventory` in the **Path** field, and then click **Save API**.
+
+    ![Resource Configuration](../assets/img/tutorials/inventory-service-api-trigger.png){.cInlineImage-half}
+    
+4. To read the information in the order, define the required parameters.
```

Comment:
The rest of the things in this step is like asking the user to follow blindly. So can we change this bit first by giving the order request sample, and saying this is how we are going to process it?